<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OptimusPrime v1</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>OptimusPrime.png</normaloff>OptimusPrime.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>181</width>
      <height>611</height>
     </rect>
    </property>
    <property name="title">
     <string>My Portfolio</string>
    </property>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>421</height>
      </rect>
     </property>
     <property name="title">
      <string>Choose Assets</string>
     </property>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>55</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="codeLineEdit">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>85</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>31</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Acct</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>115</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 255);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>25</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>91</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Code</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>62</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Item #</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>120</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showButton">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>140</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>show</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="bucketTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>151</width>
        <height>241</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="autoScrollMargin">
       <number>6</number>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>65</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>14</number>
      </attribute>
      <row/>
      <column>
       <property name="text">
        <string>Code</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="addButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resetButton">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>140</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>reset</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>161</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Optimize / Send Order</string>
     </property>
     <widget class="QPushButton" name="optButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Optimize Weights</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sendButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Order</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Import Data</string>
     </property>
     <widget class="QPushButton" name="downButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Download Data</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>9</y>
      <width>781</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string>Balance and  Holding Status</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>771</width>
       <height>65</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>120</number>
     </attribute>
     <row>
      <property name="text">
       <string>info</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Deposit (d+2)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total Purchase</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total Eval</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total Profit/Loss</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total Earning Rate(%)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Estimated Assets</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="viewButton">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>410</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>-140</y>
       <width>611</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>잔고확인</string>
     </property>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>591</width>
        <height>61</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>예수금</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>총매입</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>총평가</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>총손익</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>총수익률</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>추정자산</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>현금주문</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>414</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Live</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>771</width>
       <height>311</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>125</number>
     </attribute>
     <column>
      <property name="text">
       <string>Item Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quantity</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Purchase Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Current Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Eval Profit Loss Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Earning Rate(%)</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>470</y>
      <width>781</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Rebalance Item List</string>
    </property>
    <widget class="QTableWidget" name="tableWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>761</width>
       <height>111</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Order Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Item Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mkt/Pending</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quantity</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
